sonar-orchestrator/src/main/java/com/sonar/orchestrator/build/SonarScanner.java

```
@@ -195,6 +197,32 @@ public boolean isUseOldSonarRunnerScript() {
     return !scannerVersion().isGreaterThanOrEquals("2.6") || useOldSonarRunnerScript;
   }
 
+  /**
+   * @since 3.15 used by SQ Scanner CLI ITs
+   */
+  public SonarScanner useNative() {
+    this.classifier = determineClassifier();
+    return this;
+  }
+
+  private static String determineClassifier() {
+    if (SystemUtils.IS_OS_LINUX) {
+      return "linux";
+    }
+    if (SystemUtils.IS_OS_WINDOWS) {
+      return "windows";
+    }
+    if (SystemUtils.IS_OS_MAC_OSX) {
+      return "macosx";
+    }
+    return null;
```

Maybe throw an exception here, just in case :)